geoengine/auth.py

```diff
@@ -206,8 +173,7 @@ def get_session() -> Session:
 
 def initialize(server_url: str,
                credentials: Optional[Tuple[str, str]] = None,
-               token: Optional[str] = None,
-               admin_token: Optional[str] = None) -> None:
+               token: Optional[str] = None) -> None:
```

Comment:
so we can still use the token?

Reply:
sure, the token is just the session id which is used in the authorization header